### ISSUE TITLE
Big Sur support?

### ISSUE THREAD

**Issue by ayroblu** [openstenoproject/plover#1176]
## Summary

The dev release v4 doesn't work in big sur. My Georgi doesn't work either on the release version v3.1.1
Not sure if this is just me

**Comment by sammdot** [comment]
I had the same issue with 4.0.0.dev8+66 after upgrading to Big Sur. I had to build Plover from source to fix it. See this little doc I wrote https://plover.readthedocs.io/en/latest/from_source.html

**Comment by ayroblu** [comment]
Oh dear, so I couldn't run the app builder, it fails with this error:
![image](https://user-images.githubusercontent.com/4915682/100933679-95c80180-34e5-11eb-831d-549f909b8327.png)

I tried to use the launch method which works until I press a key on my georgi, then it throws an error
![image](https://user-images.githubusercontent.com/4915682/100933540-6a451700-34e5-11eb-95a0-05f626d1b254.png)

**Comment by sammdot** [comment]
I'm not sure what's happening with pip recently (hence the bug I made above), but someone else on the Plover Discord managed to successfully run the DMG that I built myself. Please try it and let me know if it works for you too: https://github.com/sammdot/plover/releases/tag/v4.0.0.dev8+235

**Comment by benoit-pierre** [comment]
I was going to suggest testing the build from #1170, but the artifact has been removed (older than 30days), and the macOS build is now broken thanks to pip new resolver...

**Comment by natecox** [comment]
came here to report the same issue. The fork @sammdot posted does work; however, it also extracts to twice the normal size for me (~400MB opposed to ~200MB).

**Comment by ColmanHumphrey** [comment]
This is great @sammdot thanks!

One small note: this was me here: #1131 and the same thing started happening in Big Sur!

Solution wasn't too bad: I had do delete the Plover entry from Security & Privacy > Accessibility, and then close and open the new install, to get the app to ask Mac again for access. I guess it was the prior version of plover in that slot blocking things.

**Comment by sammdot** [comment]
Good to know that my build is working for y'all as well. I'll keep the DMG up until the main repo gets an official build that works on Big Sur (or manages to build at all, for that matter #1177).

> The fork [...] extracts to twice the normal size for me (~400MB opposed to ~200MB).

I didn't notice (or care) when I released it that there was a significant size difference, but I just looked into it and I think I've found the culprit:

```
164M  plover-4.0.0.dev8+66.app/Contents/[...]/lib/python3.6/site-packages/PyQt5/Qt
338M  plover-4.0.0.dev8+235.app/Contents/[...]/lib/python3.7/site-packages/PyQt5/Qt
```

I'm not sure why the Qt upgrade (which, by the way, wasn't even a *major* version upgrade &ndash; 5.9 to 5.15) would cause it to be twice the size, but I'm inclined to keep it as it is in case removing any part of it would break some plugins.

The +235 build also contained Tcl/Tk libraries, which Plover doesn't use, but removing those only shaved off about 10 MB. 😞 Otherwise I'm not sure that there's much I can do.

**Comment by ayroblu** [comment]
@ColmanHumphrey Thanks so much! I was tearing my hair out about why it wasn't working on my georgi I did as you said, removed `Plover` (and `env`) from security accessibility and then tried to type with Plover, popped back up a security suggestion, added it back and it works again, I'm so happy, thanks so much!

Side note, rough guess on the Qt business is that naive building for arm as well as intel just doubles the size 🤷

**Comment by ColmanHumphrey** [comment]
Not sure how useful this is, but I just realised all was going well on a different laptop where I had installed Plover on Big Sur without having any issues - prior to the laptop where I looked up this thread. I didn't know how I had solved that... Until I saw I had accidentally installed v3.1.1. No idea why your Georgi doesn't work @ayroblu but mine seems to work Ok for now. Odd.

Just had to do another Big Sur install on an M1 macbook, and 3.1.1 is working with the Georgi again. 

What's the main difference between the versions? I recall switching to 4 a long time ago when there was some issue with v3
